workflows/general/alignment-post.wdl

```diff
@@ -11,7 +11,8 @@ workflow alignment_post {
         outputs: {
             processed_bam: "Input BAM after being transformed by standard processing",
             bam_index: "BAI index associated with `processed_bam`",
-            bam_checksum:  "STDOUT of the `md5sum` command run on the input BAM that has been redirected to a file"
+            bam_checksum:  "STDOUT of the `md5sum` command run on the input BAM that has been redirected to a file",
+            validate_report: "Validation report produced by `picard ValidateSamFile`. Validation warnings and errors are logged."
```

Comment:
Does this need a trailing comma?

Reply:
Huh. I was actually under the impression that a trailing comma there would cause errors. But I just double checked in both `miniwdl` and `sprocket` and they're ok with or without that comma 🤷